index.js

```diff
@@ -12,29 +14,29 @@ function _middleware (limits, fields, fileStrategy) {
 
 class Multer {
   constructor (options) {
-    this.limits = options.limits
+    this[kLimits] = options.limits
```

Comment:
out of curiosity, what's the benefit of using a  `Symbol` here?

Reply:
This was made so that the limits can't be seen/modified from the outside.

I don't really have anything against that, but if it's possible it should be documented, and since it wasn't possible before I don't think that there is a use case for it.

If we leave it in but don't document, then my fear is that we might break it in a future update without meaning to ☺️

Reply:
makes complete sense! thank you